bionemo-recipes/recipes/esm2_accelerate_te/train.py

```diff
@@ -36,6 +37,9 @@
 @hydra.main(config_path="hydra_config", config_name="L0_sanity", version_base="1.2")
 def main(args: DictConfig):
     """Entrypoint."""
+    # add wandb logging on main process
+    if not torch.distributed.is_initialized() or torch.distributed.get_rank() == 0:
+        wandb.init(config=OmegaConf.to_container(args, resolve=True, throw_on_missing=True), **args.wandb_init_args)
```

Comment:
do we need to add some basic entry to `defaults.yaml`? What happens when TrainingArguments says report-to 'wandb'? which run name takes precedence?